malduck/procmem/procmem.py

```diff
@@ -766,18 +781,19 @@ def map_offset(off, len):
             if ptr is not None and addr <= ptr < addr + length:
                 return ptr
 
-        return ruleset.match(data=self.readp(0), offset_mapper=map_offset)
+        return ruleset.match(
+            offset_mapper=map_offset, extended=extended, data=self.readp(0)
+        )
 
     def _findbytes(self, yara_fn, query, addr, length):
         if isinstance(query, bytes):
             query = query.decode()
 
         rule = Yara(strings=YaraString(query, type=YaraString.HEX))
-        match = yara_fn(rule, addr, length)
+        match = yara_fn(rule, addr, length, True)
```

Comment:
This free-hanging `True` looks really weird, are we allowed to use kwargs here or not?

Reply:
Fixed